mantidproject/mantid
# Move project loading and saving code out of ApplicationWindow

**Do not merge until #16760 has been merged**

This moves most of the code for project saving and loading out of the application window class and into a new class. More work needs to be done here. I'm wary of this ticket growing in scope so I want to pull this in the current state and then continue refinement in #16768.

**To test:**
- [x] Wait for #16760 to be merged
- Check that the `MantidPlotProjectSerialiseTest.py` tests are still passing
- Play with project saving/loading and make sure it still works. A semi comprehensive list of manually performed tests which should and shouldn't work are listed in #15602.

Fixes #16672 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
